projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -345,6 +345,23 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
      */
     public rowDimensionResizing = true;
 
+    /**
+     * @hidden @internal
+     */
+    private _emptyRowDimension: IPivotDimension;
```

Comment:
Could just be a getter, I don't think we need to set it. Something like:

```
    /**
     * @hidden @internal
     */
    private _emptyRowDimension: IPivotDimension = {memberName: '', enabled: true};
    public get emptyRowDimension(): IPivotDimension {
        return this._emptyRowDimension;
    }
```

Re-using the same object instance will allow resizing to set width to it.
You might also need to update some of the methods that get the total widths, like `pivotRowWidths` to use this instance to reflect resizing via the dimension cell correctly.